run-golangci-lint.sh

```diff
@@ -1,2 +1,2 @@
-#!/usr/bin/env bash
-exec golangci-lint run ./...
+#!/bin/sh
+golangci-lint "$@"
```

Comment:
Would it make sense to keep the `exec` and `run` here? I imagine we'll always want those

Reply:
sure